pkg/policyreport/reportcontroller.go

```diff
@@ -877,6 +883,7 @@ func (g *ReportGenerator) updateReportsForDeletedResource(resName string, new *u
 			polr.SetGroupVersionKind(schema.GroupVersionKind{Group: gv.Group, Version: gv.Version, Kind: "PolicyReport"})
 
 			if _, err := g.pclient.Wgpolicyk8sV1alpha2().PolicyReports(new.GetNamespace()).Update(context.TODO(), polr, metav1.UpdateOptions{}); err != nil {
+				g.log.Error(err, "failed to update clusterPolicyReport", "polName", polr.Name)
```

Comment:
nit: clusterPolicyReport --> policyReport

Reply:
I added that quickly for debugging, I will remove it, thanks.

Reply:
fixed